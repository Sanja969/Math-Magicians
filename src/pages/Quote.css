.Quote {
  color: #03a9f4;
}

.Quote > span {
  font-weight: 900;
}
