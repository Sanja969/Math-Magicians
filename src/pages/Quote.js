import React from 'react';
import './Quote.css';

const Quote = () => (
  <div className="pt-5 mt-5 quotes-box">
    <h3 className="Quote m-5 py-2" title="quote">
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding.
      <span>— WIlliam Paul Thurston</span>
    </h3>
    <h3 className="Quote m-5 py-2" title="quote">
      Without mathematics, there’s nothing you can do. Everything around you is mathematics.
      Everything around you is numbers.
      <span>— Shakuntala Devi, Indian writer and mental calculator</span>
    </h3>
    <h3 className="Quote m-5 py-2" title="quote">
      Pure mathematics is, in its way, the poetry of logical ideas.
      <span>— Albert Einstein, German theoretical physicist</span>
    </h3>
  </div>
);

export default Quote;
