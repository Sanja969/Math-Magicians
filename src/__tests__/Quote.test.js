import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Quote from '../pages/Quote';

describe('test the Quote components', () => {
  it('renders correctly', () => {
    const { container } = render(<Quote />);
    const qoute1 = container.querySelector('.q1');
    expect(qoute1.innerHTML).toContain(
      'Mathematics is not about numbers',
    );
  });
  it('matches the snapshot', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
