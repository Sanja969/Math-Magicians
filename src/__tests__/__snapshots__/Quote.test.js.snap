// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the Quote components matches the snapshot 1`] = `
<div
  className="pt-5 mt-5 quotes-box"
>
  <h3
    className="Quote q1 m-5 py-2"
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <span>
      — WIlliam Paul Thurston
    </span>
  </h3>
  <h3
    className="Quote q2 m-5 py-2"
  >
    Without mathematics, there’s nothing you can do. Everything around you is mathematics. Everything around you is numbers.
    <span>
      — Shakuntala Devi, Indian writer and mental calculator
    </span>
  </h3>
  <h3
    className="Quote q3 m-5 py-2"
  >
    Pure mathematics is, in its way, the poetry of logical ideas.
    <span>
      — Albert Einstein, German theoretical physicist
    </span>
  </h3>
</div>
`;
